spec/stack_spec.rb

```diff
@@ -117,6 +118,69 @@
     expect(@logger).to have_received(:info).with(/Deleting #{name} stack.../)
   end
 
+  context "#parameters_for_update" do
+    it "uses defaults, volatile, use_previous_value" do
```

Comment:
same as above, would be nice to pull these stacks out into lets, if nothing else but to make the test easier to read

Reply:
These two stack declarations need to be made in this order to simulate an update (and only used in this spec), so will leave as is.